src/node_modules/uiconfig/pallete.js

```diff
@@ -0,0 +1,30 @@
+module.exports = {
```

Comment:
this file shouldn't exist (the `uiconfig` module is meant to link configs in `/config` with the `ui` code), a better place given the content is probably `src/node_modules/ui/pallete.js`, or if we do want this to be set by the admin deploying the app then within the `ui` property in `/config`.